### DISCUSSION TITLE
Inferring species richness patterns from sdmtmb?

### DISCUSSION THREAD

**Discussion by henryhansen** [sdmTMB/sdmTMB#231] [Ideas]
Is there a straight forward way in sdmtmb to do aggregated/stacked species distribution models like shown here ([paper](https://onlinelibrary.wiley.com/doi/10.1111/geb.12102))? I know that sdmtmb is for univariate responses but with "stacking" approaches it seems possible to inform spatial and temporal shifts in species richness using single species fitted sdms. Thanks for your help.

**Comment by seananderson** [comment]
You can fit and predict separate SDMs for many species and combine the predicted values however you'd like. You could certainly follow eq. 2 in [that paper](https://onlinelibrary.wiley.com/doi/10.1111/geb.12102) and sum the site-level occurrence probabilities. We do just that in Fig. 8 of [this paper](https://cdnsciencepub.com/doi/10.1139/cjfas-2022-0108). You'll just need to make sure that you predict onto the same grid/spatial locations across species. I believe the post hoc 'adjustment' model that [Calabrese et al. (2014)](https://onlinelibrary.wiley.com/doi/10.1111/geb.12102) apply could also be done with such output.

**Reply by henryhansen** [reply]
Thanks @seananderson. I assume this functionality is not a part of the sdmtmb package already so it's nice to know that it can be done without too much hassle! Thanks again.
